app/Controllers/QueryController.js

```diff
@@ -0,0 +1,18 @@
+const { getQueryVocabulary } = require('../Services/QueryServiceProvider.js');
+const { parseTokenUserId } = require('../utils/index.js');
+
+async function sendQueryVocabulary(req, res) {
+  // get userId from request
+  const userId = await parseTokenUserId(req);
```

Comment:
Here you can use `req.user.id`. It comes from https://github.com/vocascan/vocascan-server/blob/79ea05cbc991c700186f6862bac43cd25d0d4601/app/Middleware/ProtectMiddleware.js#L33